.rubocop.yml

```diff
@@ -7,7 +7,7 @@ Metrics/CyclomaticComplexity:
 
 
 Metrics/LineLength:
-  Max: 120
+  Enabled: false
```

Comment:
Great find! I have tried this, and it doesn't seem to work. Maybe a Rubocop bug... :disappointed: